examples/llm/rag/README.md

```diff
@@ -0,0 +1,272 @@
+<!--
+SPDX-FileCopyrightText: Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+SPDX-License-Identifier: Apache-2.0
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Retrieval Augmented Generation (RAG) Pipeline
+
+## Table of Contents
+
+## Background Information
+
+### Purpose
+
+The purpose of this example is to illustrate how a user could build a Retrieval Augmented Generation pipeline
+integrating informational feeds and an LLM service into a Morpheus pipeline. This example builds on the previous
+example [completionpipeline](../completion/README.md), by adding the ability to augment LLM queries with context
+information from a knowledge base. Appending this context helps improve the responses from the LLM by providing
+additional background contextual and factual information which the LLM can pull from for its response.
+
+### Source Documents
+
+- In order for this pipeline to function correctly, a Vector Database must already have been populated with information
+  that can be retrieved.
+- An example of populating a database is illustrated in [FEA]: Create Sherlock example for VDB Upload #1298.
```

Comment:
Why reference the github issue and not a link to `examples/llm/vdb_upload/README.md`?